pkgs/applications/networking/instant-messengers/element/element-desktop.nix

```diff
@@ -8,12 +8,12 @@
 
 let
   executableName = "element-desktop";
-  version = "1.7.28";
+  version = "1.7.29";
   src = fetchFromGitHub {
     owner = "vector-im";
     repo = "element-desktop";
     rev = "v${version}";
-    sha256 = "0p88gw1y37q35qqf94w3qlb84s2shm41n1pw5fgx0c0ymlzpl636";
+    sha256 = "sha256-nCtgVVOdjZ/OK8gMInBbNeuJadchDYUO2UQxEmcOm4s=";
```

Comment:
nit: Could we keep the old hash format, please? At least until we are moving towards `hash = ....` across all of nixpkgs.

Reply:
How do you actually generate a sha256 of the old format? When I download the files and run `sha256sum` on them, I get a longer string.

Reply:
`nix to-base32 sha256-...`.

That said I don't see a real need to it since the minimum Nix version we now have in `nixpkgs` is 2.2 which supports these SRI-style hashes already

Reply:
OK. I'll resolve, as I don't think this needs to change. I grepped for `sha256-` and there are >3200 instances in the repo.